jsonschema-maven-plugin/src/main/java/com/github/victools/jsonschema/plugin/maven/SchemaGeneratorMojo.java

```diff
@@ -187,21 +186,13 @@ private void generateSchema(Class<?> schemaClass) throws MojoExecutionException
      * @throws MojoExecutionException in case of problems
      */
     private void generateSchemaForPackage(String packageName) throws MojoExecutionException {
-        Reflections reflections = new Reflections(this.getClassLoader(), packageName, new SubTypesScanner(false));
-        Set<Class<?>> subTypes;
-        try {
-            subTypes = reflections.getSubTypesOf(Object.class);
-        } catch (ReflectionsException reflectionsException) {
-            if (reflectionsException.getMessage().contains("Scanner SubTypesScanner was not configured")) {
-                throw new MojoExecutionException("Could not locate any classes in package " + packageName, reflectionsException);
-            } else {
-                throw new MojoExecutionException("Unknown problem when locating classes in package " + packageName, reflectionsException);
+        Reflections reflections = new Reflections("", new SubTypesScanner(false), this.getClassLoader());
+        Set<String> classes = reflections.getAllTypes();
```

Comment:
Do you have a rough feeling of what this means for the performance?
Would it make sense to only look-up all types once and provide them as input to the method for each `<packageName>` entry in the configuration, e.g.
- `generateSchemaForPackage(Set<String> classesOnClasspath, String packageName)`